### PR TITLE
Use README as long_description for PyPI

### PR DESCRIPTION
**Pull request by bersace**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+=========
+ Install
+=========
+
+Temboard is composed of 2 basic elements:
+
+- A lightweight **agent** that you need to install on every PostgreSQL server
+  you want to manage. See `dalibo/temboard-agent
+  <https://github.com/dalibo/temboard-agent>`_ for the agent project.
```
Comment (bersace) [inline README.rst]: cc @julmon i'm updating the README in this PR because of format change.

Comment (pgiraud) [inline README.rst]: Too bad that Pypi doesn't support Markdown.

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+   :target: https://circleci.com/gh/dalibo/temboard
+   :alt: CircleCI
+
+.. |CodeCov| image:: https://codecov.io/gh/dalibo/temboard/branch/master/graph/badge.svg
+   :target: https://codecov.io/gh/dalibo/temboard
+   :alt: Code coverage
+
+.. |Temboard| image:: https://github.com/dalibo/temboard/raw/master/doc/temboard.png
+   :target: http://temboard.io/
+   :alt: Temboard logo: a bird
```
Comment (pgiraud) [inline README.rst]: More precisely a woodpecker if I'm not wrong.

Comment (bersace) [inline README.rst]: cc @julmon

**Pull request merged**